scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -190,10 +174,6 @@ workflow Mutect2 {
         File filtered_vcf = Filter.filtered_vcf
         File filtered_vcf_index = Filter.filtered_vcf_index
         File contamination_table = Filter.contamination_table
-        Array[String] tumor_bam_sample_names = M2.tumor_bam_sample_name
```

Comment:
These are what are extracted from the bam file?  Are you sure we can delete these?  These are useful when calling from parent workflows....

Reply:
I will restore them.